ncdiff/src/yang/ncdiff/runningconfig.py

```diff
@@ -97,6 +97,11 @@
     (re.compile(r'^ *mpls mldp static '), 0),
     (re.compile(r'^ *router '), 0),
     (re.compile(r'^ *summary-address '), 1),
+    (re.compile(r'^ *mobile-network '), 4),
+    (re.compile(r'^ *ospfv3 neighbor '), 1),
+    (re.compile(r'^ *ospfv3 1 ipv(\d) neighbor'), 1),
```

Comment:
What is this `1` in `ospfv3 1 ipv4 neighbor`? Is it a process-id? I need a snip of running-config to understand its meaning.

Reply:
Hi,
This is the output for this
```
290099:  -   ospfv3 1 ipv6 neighbor FE80::AC5 priority 100 poll-interval 100 cost 100
290100:  -   ospfv3 1 ipv6 neighbor FE80::AC4 priority 1 poll-interval 1 database-filter all out
290101:  -   ospfv3 1 ipv6 neighbor FE80::AC3 priority 11 cost 1 database-filter all out
290102:  -   ospfv3 1 ipv6 neighbor FE80::AC2 poll-interval 1 cost 1 database-filter all out
290103:  -   ospfv3 1 ipv6 neighbor FE80::AC1 priority 25 cost 10
290104:  -   ospfv3 1 ipv6 neighbor FE80::AB9 priority 25 poll-interval 65535
290105:  -   ospfv3 1 ipv6 neighbor FE80::AB1 priority 10 database-filter all out
290106:  -   ospfv3 1 ipv6 neighbor FE80::AB2 poll-interval 255 database-filter all out
290107:  -   ospfv3 1 ipv6 neighbor FE80::AB3 cost 10 database-filter all out
290108:  -   ospfv3 1 ipv6 neighbor FE80::AB4 priority 255
290109:  -   ospfv3 1 ipv6 neighbor FE80::AB5 poll-interval 65535
290110:  -   ospfv3 1 ipv6 neighbor FE80::AB6 database-filter all out
290111:  -   ospfv3 1 ipv6 neighbor FE80::AB7 cost 1
```
and here is the pass log : https://earms-trade.cisco.com/tradeui/logs/logfile?logfile=TaskLog.Task-1&begin=27157921&size=853292&archive=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr%2Fusers%2Fvijviswa%2Farchive%2F24-06%2Fcli2show_job.2024Jun19_08:53:05.637932.zip&ats=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr&submitter=vijviswa&from=trade&view=all&atstype=pyATS